### DISCUSSION TITLE
Social Media?

### DISCUSSION THREAD

**Discussion by Maxog3n** [titinko/utsu#66] [Ideas]
I remember in your original post on the Utau forums you were finding a way to market Utau. In my opinion, setting up an account on a website like Twitter would be a good way to get it out there!

**Comment by titinko** [comment]
Twitter is not my strong point, but I was thinking of maybe making a Discord around the time of the next release!  Since there are several people contributing to the software now it would be a nice way to help everyone coordinate.

**Reply by Maxog3n** [reply]
That's an awesome idea!

**Reply by pseudo-producer** [reply]
Never thought of creating a fanpage? Here in Brazil, we often use facebook and twitter to communicate. But I confess that the Brazilian fundon of UTAU has a solid group in Discord ...

**Comment by Mubarik1122** [comment]
I'm social media managing expert ...is there any website where i found best jobs...?
